src/main/java/edu/cornell/kfs/concur/service/impl/ConcurEventNotificationConversionServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package edu.cornell.kfs.concur.service.impl;
+
+import java.text.ParseException;
+
+import org.kuali.rice.core.api.datetime.DateTimeService;
+
+import edu.cornell.kfs.concur.businessobjects.ConcurEventNotification;
+import edu.cornell.kfs.concur.rest.xmlObjects.ConcurEventNotificationDTO;
+import edu.cornell.kfs.concur.service.ConcurEventNotificationConversionService;
+
+public class ConcurEventNotificationConversionServiceImpl implements ConcurEventNotificationConversionService {
+    static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ConcurEventNotificationConversionServiceImpl.class);
+    
+    protected DateTimeService dateTimeService;
+
+    @Override
+    public ConcurEventNotification convertConcurEventNotification(ConcurEventNotificationDTO concurEventNotificationDTO) {
+        ConcurEventNotification concurEventNotification = new ConcurEventNotification();
+        concurEventNotification.setContext(concurEventNotificationDTO.getContext());
+        try {
+            concurEventNotification.setEventDateTime(dateTimeService.convertToSqlDate(concurEventNotificationDTO.getEventDateTime()));
+        } catch (ParseException e) {
+            LOG.warn("Incoming EventDateTime was invalid and it was reset to the current date");
+            concurEventNotification.setEventDateTime(dateTimeService.getCurrentSqlDate());
```

Comment:
I changed the code to throw the ParseException. when the end point calls this conversion method, if a ParseException occurs I throw a javax.ws.rs.BadRequestException. This will return a n HTTP 400. Since a valid date is expected in the incoming xml I think a 400 error would be appropriate in this case.